target/linux/imx/cortexa7/config-default

```diff
@@ -1,27 +1,35 @@
 CONFIG_ARM_CPU_SUSPEND=y
 CONFIG_ARM_GIC=y
-CONFIG_CLK_IMX6UL=y
+CONFIG_ARM_PSCI=y
+CONFIG_ARM_PSCI_FW=y
+CONFIG_AT803X_PHY=y
+CONFIG_CLK_IMX7D=y
 CONFIG_CMA=y
 CONFIG_CMA_AREAS=7
 # CONFIG_CMA_DEBUG is not set
 # CONFIG_CMA_DEBUGFS is not set
 # CONFIG_CMA_SYSFS is not set
 CONFIG_CONTIG_ALLOC=y
 # CONFIG_DMA_CMA is not set
+CONFIG_FEC=y
 CONFIG_GENERIC_IRQ_EFFECTIVE_AFF_MASK=y
 CONFIG_HW_RANDOM_IMX_RNGC=y
 CONFIG_JFFS2_FS=y
 CONFIG_LEDS_GPIO=y
 CONFIG_MEMORY_ISOLATION=y
+CONFIG_MMC_SDHCI_ESDHC_IMX=y
+CONFIG_MMC_SDHCI_IO_ACCESSORS=y
+CONFIG_MMC_SDHCI_OF_ESDHC=y
+# CONFIG_MMC_SDHCI_PCI is not set
 CONFIG_MTD_SPI_NOR=y
 CONFIG_MTD_SPLIT_FIRMWARE=y
 CONFIG_MTD_SPLIT_FIT_FW=y
-CONFIG_PINCTRL_IMX=y
-CONFIG_PINCTRL_IMX6UL=y
+CONFIG_NOP_USB_XCEIV=y
 CONFIG_PM_GENERIC_DOMAINS=y
 CONFIG_PM_GENERIC_DOMAINS_OF=y
-CONFIG_SOC_IMX6=y
-CONFIG_SOC_IMX6UL=y
```

Comment:
@Leo-PL The i.MX6UL/L series is based on the Cortex-A7, I'm planning to include several boards within this subtarget. Any reason why did you remove these configs?

Reply:
I wasn't aware of that only until recently - of course this should stay, at least CONFIG_SOC_IMX6UL for sure.

Reply:
I restored all i.MX6UL-related configs back - will post update soon, after I rebuild and test.